source/Symbol/SwiftASTContext.cpp

```diff
@@ -3432,11 +3433,14 @@ swift::ASTContext *SwiftASTContext::GetASTContext() {
 
   // 2. Create and install the module interface loader.
   //
-  // TODO: It may be nice to reverse the order between MIL and SML in
-  //       LLDB, since binary swift modules likely contain private
-  //       types that the module interfaces are missing. On the
-  //       other hand if we need to go looking for a module on disk,
-  //       something is already screwed up in the debug info.
+  // The ordering of 2-4 is the same as the Swift compiler's 1-3,
```

Comment:
For swiftc, ModuleInterfaceLoader is number 3 and ClangImporter is number 5. We might have the SourceLoader and (MemoryBufferSerializedModuleLoader for lldb) come before.